app/controllers/spree/admin/gdpr_requests_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Spree
+  module Admin
+    class GdprRequestsController < ResourceController
+      def index
+        respond_with(@collection)
+      end
+
+      private
+
+      def collection
+        @collection ||=
+          Spree::GdprRequest
+            .order(served_at: :asc, created_at: :desc)
```

Comment:
If you use `ASC` ordering, null values will come after any non-null values (unless you specify `NULLS FIRST`). This is not what we want: unserved requests should come _before_ any served requests, so that admins can find them more easily.

See https://www.postgresql.org/docs/8.3/queries-order.html for reference:

> The `NULLS FIRST` and `NULLS LAST` options can be used to determine whether nulls appear before or after non-null values in the sort ordering. By default, null values sort as if larger than any non-null value; that is, `NULLS FIRST` is the default for `DESC` order, and `NULLS LAST` otherwise.